### PR TITLE
fix: imports in generator

### PR DESCRIPTION
**Pull request by anshumanv**
**What kind of change does this PR introduce?**

fix

**Did you add tests for your changes?**
Yes

**If relevant, did you update the documentation?**
NA

**Summary**
- No not import from webpack-cli, use instance
- 
**Does this PR introduce a breaking change?**

no

**Other information**

### CODE REVIEW

File: packages/generators/src/addon-generator.ts [hunk 1]
```diff
@@ -37,6 +33,17 @@ const addonGenerator = (
     templateFn: (instance: any) => Record<string, unknown>,
 ): Generator.GeneratorConstructor => {
     return class extends Generator {
+        public logger: Record<string, (string) => void>;
+        public getPackageManager: (val: void) => string;
```
Comment (alexander-akait) [inline packages/generators/src/addon-generator.ts:37]: let's move it under `utils`

Comment (alexander-akait) [reply packages/generators/src/addon-generator.ts:37]: so we can add more utils in future

Comment (anshumanv) [reply packages/generators/src/addon-generator.ts:37]: yep lets do it

Comment (anshumanv) [reply packages/generators/src/addon-generator.ts:37]: need to union types for all utils, it is nightmare 😞

Comment (alexander-akait) [reply packages/generators/src/addon-generator.ts:37]: You can use `any` right now, after migrate on ts or jsdocs we will fix it

Comment (anshumanv) [reply packages/generators/src/addon-generator.ts:37]: thanks

**Review by alexander-akait — APPROVED** [review]

**Pull request merged**